### PR TITLE
Fixing sphinx warnings

### PR DESCRIPTION
**Pull request by AndreiSavici**
Fixing some sphinx warnings.
Added figure for release notes from 3.8 branch
I changed the links for renamed algorithms to point to the new version

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?

- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/interfaces/Tomographic_Reconstruction.rst [hunk 1]
```diff
@@ -347,8 +347,10 @@ You can run local reconstructions as well, however that requires properly settin
 - External python interpretor path, for example:
   - C:/Anaconda/python.exe for Windows
```
Comment (peterfpeterson) [inline docs/source/interfaces/Tomographic_Reconstruction.rst]: These should be a sub-list and that still isn't rendering correctly. Also backtick the directories

File: docs/source/interfaces/Tomographic_Reconstruction.rst [hunk 2]
```diff
@@ -347,8 +347,10 @@ You can run local reconstructions as well, however that requires properly settin
...
   - ~/Anaconda2/bin/python for Linux
+
 - PYTHONPATH environment variable should contain the Python directories 
-that have the installed plugins for the supported tools
+  that have the installed plugins for the supported tools
+
 - (Optional) Scripts directory
   - By default this will be properly setup from the Mantid installation
```
Comment (peterfpeterson) [inline docs/source/interfaces/Tomographic_Reconstruction.rst]: Should be a sub-list that isn't rendering correctly

File: docs/source/release/v3.8.0/framework.rst [hunk 3]
```diff
@@ -91,7 +91,7 @@ Improved
 
 - :ref:`FindSXPeaks <algm-FindSXPeaks>`: Fixed a bug where peaks with an incorrect TOF would stored for some intrument geometries.
 
-- :ref:`LoadILL <algm-LoadILL>` was renamed to `LoadILLTOF <algm-LoadILLTOF>` to better reflect what it does. The new algorithm can also handle cases where the monitor IDs are greater than the detector IDs.
+- :ref:`LoadILL <algm-LoadILLTOF>` was renamed to `LoadILLTOF <algm-LoadILLTOF>` to better reflect what it does. The new algorithm can also handle cases where the monitor IDs are greater than the detector IDs.
```
Comment (peterfpeterson) [inline docs/source/release/v3.8.0/framework.rst]: second link needs a `:ref:` to make it work. Same for `ConvertUnitsUsingDetectorTable` on line 78 above.

**Comment by peterfpeterson** [comment]
Was 65 warnings, now it is only 64 (squashed a missing img issue).

**Review by peterfpeterson — CHANGES_REQUESTED** [review]

**Comment by AndreiSavici** [comment]
I have only one warning, and that is because I am using a newer version of sphinx, where `pngmath` is deprecated
```
Running Sphinx v1.4.8
WARNING: sphinx.ext.pngmath has been deprecated. Please use sphinx.ext.imgmath instead.
```

**Review by peterfpeterson — APPROVED** [review]

**Comment by peterfpeterson** [comment]
I'm down to the same warning. :shipit:

**Pull request merged**